python/ray/tune/experiment/trial.py

```diff
@@ -799,13 +799,15 @@ def init_local_path(self):
             self.relative_logdir = _create_unique_logdir_name(
                 str(self.local_experiment_path), self._generate_dirname()
             )
-        assert self.logdir
-        logdir_path = Path(self.logdir)
-        if _is_path_too_long(logdir_path):
+        assert self.local_path
+        logdir_path = Path(self.local_path)
+        max_path_length = os.pathconf("/", "PC_PATH_MAX")
```

Comment:
I think this function is not available on Windows.

Reply:
I see - re-introduced a function for this